src/main/java/alison/task/Deadline.java

```diff
@@ -0,0 +1,46 @@
+package alison.task;
+
+import alison.exception.AlisonException;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+
+    private String by;
```

Comment:
Would renaming the field `by` to `deadline` make this field more easily understood?